features/bevragen/protocollering.feature

```diff
@@ -0,0 +1,85 @@
+#language: nl
+
+Functionaliteit: Als burger wil ik zien wie welke gegegevens van mij heeft gezien of gebruikt
+  Zoeken en raadplegen van gegevens van burgers worden "geprotocolleerd" (formeel gelogd).
+
+  Protocollering moet gebeuren op twee niveau's:
+  - bij de RvIG op het niveau van afnemende organisaties
+  - bij de afnemende organisatie op het niveau van gebruikers
+
+  Deze feature beschrijft alleen de werking van protocollering op het niveau van afnemende organisaties bij de bron (RvIG)
+
+  Daarin worden de volgende gegevens vastgelegd:
+  - request_id: unieke identificatie van de berichtuitwisseling waarmee het protocolleringsrecord te koppelen is aan andere logging van de API
```

Comment:
Waar komt zo'n request_id eigenlijk vandaan wordt dat bij logging gegenereerd door de database en kan ik het zien als een key of wordt deze uit het bericht gehaald?

In dat laatste geval zou je bij protocollering op de 2 niveaus hetzelfde request_id kunnen gebruiken.
Dan zou je de logging op het niveau van de gebruikers kunnen koppelen aan de logging op het niveau van afnemende organisatie. Ik heb echter geen idee of dat van belang is.

Reply:
er is niet gespecificeerd hoe de id moet worden bepaald. is implementatie. moet alleen uniek zijn